seed/migrations/0086_auto_20180111_1531.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.6 on 2018-01-11 23:31
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def convert_rule_units_to_pint(apps, schema_editor):
+    """
+    pre-pint rules (esp old defaults) will have their units specified as
+    human-readable strings instead of Pint specs. This converts them so
+    that any custom rules can be maintained instead of just dumped and
+    re-created.
+    """
+    Rule = apps.get_model('seed', 'Rule')
+    for rule in Rule.objects.all():
+        if rule.units == 'square feet':
+            rule.units = 'ft**2'
+        elif rule.units == 'kBtu/sq. ft./year':
+            rule.units = 'kBtu/ft**2/year'
```

Comment:
hmm. we should really just update the old units with the pint version of the units. save it for another day.